packages/graphback/src/resolvers/knex/BaseKnexResolver.ts

```diff
@@ -87,7 +87,7 @@ public deletedSub = (typeName: string): string => {
 public typeRelation = (relation: string, columnName: string, fieldName: string, tableName: string): string => {
   if(relation === 'OneToOne') {
     return `${fieldName}: (parent: any, _: any, context: GraphQLContext) => {
-      return context.db.select().from('${tableName}').where('${columnName}', '=', parent.id)
+      return context.db.select().from('${tableName}').where('id', '=', parent.${fieldName}Id)
```

Comment:
parent might not have a `${fieldName}Id` always

Reply:
If it has a OneToOne relationship it should always have `${fieldName}Id`.

That's what defines the OneToOne relationship. What other circumstance could there be to have a OneToOne relationship and NOT have `${fieldName}Id`?

Maybe I'm missing something? All of the scenarios worked `notes.user`

Reply:
If relationships are defined in the `.graphql` file using `user: User!` and `comments: [Comment!]!` constructs then are OneToOne and OneToMany directives still needed?